rasa/utils/tensorflow/model_data_utils.py

```diff
@@ -63,8 +68,20 @@ def featurize_training_examples(
                     attribute, featurizers
                 )
         output.append(attribute_to_features)
-
-    return output
+    # get sparse feature sizes
+    sparse_attributes = []
+    for attr, features in output[0].items():

```

Comment:
Also avoid using shorthand variable names. For example use `attribute` instead of `attr`.